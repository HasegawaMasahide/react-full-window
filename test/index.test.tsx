import { shallow } from "enzyme";
import React from "react";

import FullWindow from "../src";

test("component should be render", () => {
    const fw = shallow(<FullWindow />);

    // Interaction demo
    expect(fw.text()).toEqual("");
    expect(fw.find(".react-full-window")).toBeTruthy();

    // Snapshot demo
    expect(fw).toMatchSnapshot();
});
