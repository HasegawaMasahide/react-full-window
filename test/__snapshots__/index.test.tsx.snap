// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component should be render 1`] = `
<div
  className="react-full-window"
/>
`;
